src/cosalib/kubevirt.py

```diff
@@ -12,7 +34,10 @@ def kubevirt_run_ore_replicate(*args, **kwargs):
 
 
 def kubevirt_cli(parser):
-    """
-    Extend a parser with the KubeVirt options
-    """
+    parser.add_argument("--name",
+                        help="Name to append to the repository (e.g. fedora-coreos). Defaults to the build name.")
+    parser.add_argument("--repository", help="repository to push to (e.g. quay.io or quay.io/myorg)", required=True)
+    parser.add_argument('--dry-run', dest='dry_run', action='store_true')
+    parser.add_argument('--no-dry-run', dest='dry_run', action='store_false')
```

Comment:
Any reason for both? I don't think we do that for any other command.

Reply:
> Any reason for both?

Removed `--dry-run`. Did not intent to keep that. :+1: 

>I don't think we do that for any other command.

Do you think I should remove the whole dry-run functionality? I found it very helpful during debugging, since I could run pretty much everything during development.